commondata/LEP_eeWW_182GeV.yaml

```diff
@@ -1,4 +1,11 @@
 dataset_name: LEP_eeWW_182GeV
+doi: 10.1016/j.physrep.2013.07.004
+location: Tab. 5.6
+arxiv: 1302.3415
+hepdata: none
+units: pb
+description: Combined W− differential angular cross-section in the 10 angular bins
+luminosity: 0.16390 # fb^-1
```

Comment:
```suggestion
luminosity: 0.16390 # pb^-1
```

Reply:
163.90 pb^-1 = 0.16390 fb^-1 and the units of the lumi entry are fb^-1 everywhere